core/i18n/src/ITwinLocalization.ts

```diff
@@ -6,17 +6,16 @@
  * @module Localization
  */
 
-import i18next, { i18n, InitOptions, Module, TOptionsBase } from "i18next";
+import i18next, { i18n, InitOptions, Module } from "i18next";
 import i18nextBrowserLanguageDetector, { DetectorOptions } from "i18next-browser-languagedetector";
-import { BackendOptions } from "i18next-http-backend";
-import XHR from "i18next-xhr-backend";
+import Backend, { BackendOptions } from "i18next-http-backend";
```

Comment:
could this be dynamically imported? id hate to pull this into a frontend for no reason